File: Prog_Estruturada/Operações/meuMath.c
#include "meuMath.h"
#include<stdio.h>
#include<stdlib.h>

char *toString(dados x){
  char *result = (char *)malloc(250);
  sprintf(result, "Sendo a = %d e b = %d, tem se: \nsoma = %d\nsubtracao = %d\nmultiplicacao= %d\ndivisao inteira = %d\ndivisao real = %.2f\n\n", x.a,x.b,soma(x),subtracao(x),mult(x),divInteiro(x), divReal(x) );
  return(result);
}
dados entrada(){
  dados result;
  printf("a: ");
  scanf("%d", &result.a);

  printf("b: ");
  scanf("%d", &result.b);
  return result;
}
int soma(dados x){
  return (x.a + x.b);
}
int subtracao(dados x){
  return (x.a - x.b);
}
int mult(dados x){
  return (x.a * x.b);
}
int divInteiro(dados x){
  return (x.a /x.b);
}
float divReal(dados x){
  return (float)(x.a / (float)x.b);
}
//programação estruturada

File: POO (COMLPLETO)/POO/src/POO/MeuMath.java
package POO;

public class MeuMath {
  private int a;
  private int b;

  public MeuMath(int a, int b) {
    this.a = a;
    this.b = b;
  }

    public int getA() {
        return a;
    }

    public void setA(int a) {
        this.a = a;
    }

    public int getB() {
        return b;
    }

    public void setB(int b) {
        this.b = b;
    }
    
    public int soma() {
      return(a + b);
    }
    
    public int subtracao() {
      return(a - b);
    }
    
    public int mult() {
      return(a * b);
    }
    
    public int divInteira() {
      return(a / b);
    }
    
    public double divReal() {
      return(a / (double)b);
    }
    
    public String toString() {
      String result;

      result = String.format("Sendo a = %d e b = %d, tem-se:\n" +
                   "Soma = %d\n" +
                   "Subtracao = %d\n" +
                   "Multiplicacao = %d\n" +
                   "Divisao Inteira = %d\n" +
                   "Divisao Real = %.2f\n",
                   a, b, soma(), subtracao(), mult(),
                   divInteira(), divReal());  
      
      return(result);
    }

}
